inspector/src/components/sourceEditor/shaderSourceEditor.ts

```diff
@@ -0,0 +1,139 @@
+import { ShaderMaterial } from "babylonjs/Materials/shaderMaterial";
+import { renderSSE, ShaderSourceRecompileCallback, HistoryDataSource } from './shaderSourceEditorUI';
+
+class Popup {
```

Comment:
We should have it already as we have a popup feature for the AnimationCurve editor